### PR TITLE
include test requirements in local.txt

### PR DESCRIPTION
**Pull request by jimabramson**
@clintonb @rlucioni

tiny change that removes a step from first-time development environment setup.

### CODE REVIEW

**Comment by rlucioni** [comment]
@jimabramson could you also modify the `make` targets, removing the `dev_requirements` and `test_requirements` targets and replacing them with a single `requirements` target that installs the contents of `local.txt`?

**Comment by clintonb** [comment]
:+1:

**Comment by jimabramson** [comment]
@rlucioni I implemented the Makefile changes.

**Comment by rlucioni** [comment]
Looks like you need to update `.travis.yml` to use the new `make` target. Once that's done and tests pass, :+1: . Thanks for making the change!

**Comment by jimabramson** [comment]
sigh, looks like we can't do local.txt in travis, so will have to do some more rejiggering.

File: Makefile [hunk 1]
```diff
@@ -3,19 +3,18 @@ help:
...
+	@echo '    make clean                        delete generated byte code and coverage reports        '
+	@echo '    make test_python                  run unit tests with migrations disabled                '
+	@echo '    make html_coverage                generate and view HTML coverage report                 '
+	@echo '    make quality                      run pep8 and pylint                                    '
+	@echo '    make validate                     run unit tests, followed by quality checks             '
+	@echo '    make accept                       run acceptance tests                                   '
 	@echo '                                                                                             '
 
-dev_requirements: test_requirements
+requirements: test_requirements
```
Comment (rlucioni) [inline Makefile:18]: I would just get rid of the `test_requirements` target altogether. There's no need to keep it around if we don't intend for people to be using it by itself.

Comment (jimabramson) [inline Makefile:18]: problem is travis needs a target to use.   unless you think it's better for travis to use pip directly.

Comment (rlucioni) [inline Makefile:18]: Resolved offline; Travis can't install editable sources. This is yet another reason to turn the extensions repo into an app in the ecommerce project.

**Comment by rlucioni** [comment]
:+1: (again)

**Pull request merged**